.github/workflows/check_tra.yaml

```diff
@@ -0,0 +1,33 @@
+name: Build markdowns
+
+on:
```

Comment:
Make this also a check on opened PR

Reply:
Good point. Merging master to a PR's branch is an even better example.